### ISSUE TITLE
Some typos to report, plus a suggestion for layout change

### ISSUE THREAD

**Issue by Hezkezl** [VarsityPhysicist/FFXIVRelicTracker#6]
1. In the HW View, Sharpened menu there's a typo. The full text is: 1 Singing Cluster is awarded by the waily quest 'Cut from a Different Cloth', started by Angelet (Idyllshire)   (should obviously be 'daily quest', not 'waily')
2. Also suggest changing on the same screen: "Quest objective is to complete a Daily Roulette: Leveling three times" to just "Quest objective is to complete Daily Roulette: Leveling three times."  (remove the 'a' before the duty roulette part)
3. In the 'Lux' section, there's some more missing text. "With your  equipped, complete the following trials:" (something like "completed weapon" is missing between 'your' and 'equipped')
4. In the ShB View, the Resistance tab has an errant ' showing up at the end of the top part. (The first Resistance weapon is received upon completion of a quest chain starting with:')
5. Also in the ShB view, the 'Recollection' tab is missing an item name in the following sentence: "Collect 6  from one of the following sources:" (there's a double space after the '6' there, showing there's meant to be something like 'Bitter Memory of the Dying' in that spot.)
6. Minor typo in the ARR View, Atma tab. "After collecting 1 of each atma below, speak with Jalzahn to receive an Atma weapon" the first "atma" should be capitalized, since it's capitalized in the sentence just above ("Any job may be used to acquire Atma")
7. Same tab, should put a space after the : and before the number in the line "Total of each Atma required for all remaining jobs". With 0 remaining, it looks like this: Total of each Atma required for all remaining jobs:0
8. EDIT: Another typo to add. In the HW View, the Animated tab has {Azys Lla} in brackets instead of parenthesis. (Everywhere else Azys Lla is in parenthesis, so should change to match standard)
9. another typo: The 'Complete' tab for HW View. Quest name is incorrect (says 'Some Assebly Required', but that should be 'Some Assembly Required')

Also, would it be possible to reverse the way the table looks on the 'Summary' page? Right now, the jobs scroll across the top as headers of the table, with the different relic stages as the rows underneath them. But having that reversed, with the relic stages as the headers and having the jobs stretched out underneath them, might look a little bit better. Though if you do that, you'll probably want to do that with all tabs (ARR View, ShB view, etc) to make it match...and that might be better in the long run, since the ShB view is quiiiiite wide with all the possible jobs stretched across it, and having a scrollbar on the side is more natural for people instead of having one on the bottom.

EDIT2: Quick other question I had when I was killing stuff for ARR relics just now. Would it be possible to add multiple circles showing an enemy's location for the Atma books? At the moment, there's just one red circle showing an area roughly where they are, but it would be nice if there could be multiple dots showing their exact spots. Like for Outer La Noscea, there are a few diff mineshafts some enemies can be down and they're not always clumped together in a small circle. I'd be willing to provide the data if it's possible to have multiple circles. But if it's not possible, then disregard this suggestion :)

**Comment by VarsityPhysicist** [comment]
Thank you for identifying those typos, I have corrected them

Also, I implemented a ComboBox for specifying the layout being "Horizontal" or "Vertical" for displaying stages as columns instead of rows

As to edit2: That should be doable if you want to gather the data. I don't believe there was anything limiting the # of creature circles on the map, I have it programmed to put like up to 3 or so on the map, just because that was around the max that was to be on the map at once. It shouldn't be a problem to either add more circles, or smaller targeted circles where there is data. I just went with the x,y coordinates on the wiki so that's why they are what they are now
